src/Repository/DefaultUnleashProxyRepository.php

```diff
@@ -153,16 +153,16 @@ private function addQuery(string $url, string $query): string
      *             value: string
      *         }
      *     },
-     *     impression_data: bool
+     *     impressionData: bool
      * }|null
      */
     private function validateResponse(array $response): ?array
     {
-        if (!isset($response['name'], $response['enabled'], $response['variant'], $response['impression_data'])) {
+        if (!isset($response['name'], $response['enabled'], $response['variant'], $response['impressionData'])) {
             return null;
         }
 
-        if (!is_string($response['name']) || !is_bool($response['enabled']) || !is_bool($response['impression_data']) || !is_array($response['variant'])) {
+        if (!is_string($response['name']) || !is_bool($response['enabled']) || !is_bool($response['impressionData']) || !is_array($response['variant'])) {
```

Comment:
This will fail for older versions which have `impression_data`.

Reply:
[c9568bc](https://github.com/Unleash/unleash-client-php/pull/227/commits/c9568bc449897ee8a5f3bd9df3cdd6281006ee60) this should fix it but made me think, maybe `validateResponse` should decide on a format an adapt the input into a more predictable output (i.e. always camelCase in the output despite the input format)